### ISSUE TITLE
Unable to inject service in mail views

### ISSUE THREAD

**Issue by etiennecl** [jamesmh/coravel#203]
**Describe the bug**
I've been trying to inject the IStringLocalizer inside our coravel mail templates. But it always throws because it looks like the RazorRenderer doesn't seem to be able to inject any service inside a View. I've been trying to inject other services without success. The services have been registered in the `Startup.ConfigureServices` as I'm able to inject them inside my controllers or any other class.

**Affected Coravel Feature**
Mailer

**Expected behaviour**
I would of liked for the RazorRenderer to be able to inject services in order to provide localized mail views.

**Comment by jamesmh** [comment]
Yes, that's isn't supported right now. Would require some work since the razor renderer isn't tied to the main DI container.
